jako/distribute/DistributedScan.py

```diff
@@ -91,9 +90,10 @@ def __init__(self,
 
         arguments_dict = self.__dict__
         remove_parameters = ["x", "y", "model"]
-        arguments_dict = {k: v for k, v in arguments_dict.items() if k not in remove_parameters}
+        arguments_dict = {k: v for k, v in arguments_dict.items()
+                          if k not in remove_parameters}
 
-        self.file_path = 'tmp/scanfile_remote.py'
+        self.file_path = '/tmp/scanfile_remote.py'
 
         self.save_timestamp = time.strftime('%D%H%M%S').replace('/', '')
```

Comment:
Have look in here https://github.com/autonomio/talos/blob/f68c057063bef07357b068b0eb040dc4a63c4fa0/talos/logging/logging_run.py#L9 for the correct format for timestamp

Reply:
Hi, this timestamp object is used to find out the latest csv, hence this format.